README.md

```diff
@@ -133,7 +116,106 @@ include the `restPort` configuration to tell the exporter which port to use.
+
+One way to use the exporter is by creating a WAR file with a default configuration and deploying it to a WebLogic Server instance.
+
+## Setting the configuration
+
+The web application has a main landing page, which displays the current [configuration](#configuration) and allows
+you to change it, either by uploading a replacement or an addition to the queries specified with the current one.
+Metrics will then be available from `<application-root>/metrics`.
+
+
+## Downloading the release
+
+You can find all the releases on the [Releases page](https://github.com/oracle/weblogic-monitoring-exporter/releases/).
+
+To download the web application `wls-exporter.war` and put your configuration file into the WAR, download the `getXXX.sh` script, which is provided with each release and also can be downloaded from the Releases page, and then run:
+
+```
+bash getXXX.sh <your-config-file>
+```
+
+## Building from source
+
+Use `mvn install` to build the web application. This will create `wls-exporter-<version>`, where _version_
+is the Maven-assigned version number.
+
+Adding `-Dconfiguration=<some-config-file>` will insert the specified configuration as its default and remove
+the version number to simplify deployment to WebLogic Server.
+
+# Sidecar
+
+The sidecar is a standalone process that runs the exporter. It is primarily intended for use by the
+[WebLogic Kubernetes Operator](https://github.com/oracle/weblogic-kubernetes-operator/).
+
+## Build and run with Maven
+
+There are two ways to build the sidecar implementation. The first is with Maven, using `mvn install`. Note that this
+requires JDK11 or later. Building the project with JDK8 will skip the sidecar module. The alternative is to
+[build with Docker](#building-a-docker-image) 
+
+After building, run:
+```
+java -jar wls-exporter-sidecar/target/wls-exporter-sidecar.jar
+```
+
+This will start the exporter on port 8080, and expect to find a WebLogic Server instance locally, listening on port 7001.
+
+That may be changed by specifying parameters on the command line:
+
+Setting | Default | Property
+------------ | ------------- | -------------
+Domain name | (use WLS definition) | `DOMAIN`
+Exporter port | `8080` | `EXPORTER_PORT`
+WebLogic host | `localhost` | `WLS_HOST`
+WebLogic port | `7001` | `WLS_PORT`
+Use https | `false` | `WLS_SECURE`
+
+## Configure the exporter
+
+The sidecar does not have a landing page. Configuration is done by sending a PUT request to the path `/configuration`. 
+You can do this with `curl`; when used with the [WebLogic Kubernetes Operator](https://github.com/oracle/weblogic-kubernetes-operator/), 
+the operator will automatically configure the exporter. 
+
+```
+curl -X PUT -i -u myname:mypassword \
+    -H "content-type: application/yaml" \
+    --data-binary "@<path to yaml>" \
+    http://localhost:8080/configuration
+``` 
+
+where you should replace `myname` and `mypassword` with the credentials expected by WebLogic Server for its REST API,
```

Comment:
Not sure if you got this edit:
where you should replace -> Replace, or You should replace

Reply:
I'd obviously missed it.